plugins/early_talker.js

```diff
@@ -64,3 +76,56 @@ exports.early_talker = function(next, connection) {
 
     setTimeout(function () { check(); }, pause);
 };
+
+
+/**
+ * Check if an ip is whitelisted
+ *
+ * @param  {String} ip       The remote IP to verify
+ * @return {Boolean}         True if is whitelisted
+ */
+exports.ip_in_list = function (ip) {
+    var plugin = this;
+    var ipobj = ipaddr.parse(ip);
```

Comment:
Why parse the IP if plugin.whitelist doesn't exist?   (shouldn't that check come before this one?)